junction/profiles/views.py

```diff
@@ -22,3 +26,15 @@ def dashboard(request):
                 conf_proposals[conf.name] = [proposal]
     return render(request, 'profiles/dashboard.html',
                   {'conf_proposals': conf_proposals})
+
+
+@login_required
+def profile(request):
+    if request.method == "POST":
+        city = request.POST.get('city')
+        contact_no = request.POST.get('contact_no')
+        Profile.objects.create(user=request.user, city=city, contact_no=contact_no)
```

Comment:
Whenever the user hits the endpoint, the profile is created. Please check profile table before creating a new profile.